pavelib/prereqs.py

```diff
@@ -33,7 +33,7 @@
 
 # Developers can have private requirements, for local copies of github repos,
 # or favorite debugging tools, etc.
-PRIVATE_REQS = 'requirements/private.txt'
+PRIVATE_REQS = 'requirements/adg/base.txt'
 if os.path.exists(PRIVATE_REQS):
     PYTHON_REQ_FILES.append(PRIVATE_REQS)
```

Comment:
I think we need to add `requirements/adg/testing.txt` as well

Reply:
I did, but @ahmed-faraz46 told me we don't want this here in ADG so now I removed it.